packages/driver/src/cypress/runner.ts

```diff
@@ -1492,7 +1492,7 @@ export default {
           // attach error right now
           // if we have one
           if (err) {
-            const PendingErrorMessages = ['sync skip', 'async skip call', 'async skip; aborting execution']
+            const PendingErrorMessages = ['sync skip', 'sync skip; aborting execution', 'async skip call', 'async skip; aborting execution']
```

Comment:
This test case in our repo produced this error: https://github.com/cypress-io/cypress/blob/develop/packages/driver/cypress/e2e/cy/skipping_async.cy.js#L36